elements/pfe-card/test/pfe-card.spec.ts

```diff
@@ -80,302 +56,142 @@ describe('<pfe-card>', function() {
     expect(cardFooterSlot.assignedNodes().length).to.equal(1);
   });
 
-  it(`should render a header and footer when content for those deprecated slots are added dynamically`, async function() {
-    const element = await createFixture<PfeCard>(dynamicHeaderFooter);
-
-    const header = document.createElement('h2');
-    header.setAttribute('slot', 'pfe-card--header');
-    header.textContent = 'Card Header';
+  describe('size', () => {
+    describe('is unset', () => {
+      it('should have default sizing for card parts', async function() {
+        const element = await createFixture<PfeCard>(TEMPLATES.card1);
```

Comment:
@hellogreg which accessibility-related things should we be testing here?

please see https://modern-web.dev/docs/test-runner/commands/#accessibility-snapshot for an idea of what we can do - basically get a snapshot of the AOM for a given element then assert things on in

Reply:
sure thing. please schedule a call to go over the a11y snapshot facility at your convenience